tests/framework/cli/test_starters.py

```diff
@@ -933,3 +951,40 @@ def test_invalid_add_ons(self, fake_kedro_cli):
             "Please select valid options for add-ons using comma-separated values, ranges, or 'all/none'.\n"
             in result.output
         )
+
+
+@pytest.mark.usefixtures("chdir_to_tmp")
+class TestAddOnsFromCLI:
+    @pytest.mark.parametrize(
+        "add_ons",
+        [
+            "lint",
+            "test",
+            "log",
+            "docs",
+            "data",
+            "none",
+            "test,log,docs",
+            "test,data,lint",
+            "log,docs,data,test,lint",
+            "all",
+        ],
+    )
+    def test_valid_add_ons(self, fake_kedro_cli, add_ons):
+        result = CliRunner().invoke(
+            fake_kedro_cli,
+            ["new", "--addons", add_ons],
+            input=_make_cli_prompt_input_without_addons(),
+        )
+        add_ons = _convert_addon_names_to_numbers(selected_addons=add_ons)
+        _assert_template_ok(result, add_ons=add_ons)
+        _assert_requirements_ok(result, add_ons=add_ons)
```

Comment:
```suggestion
        _assert_requirements_ok(result, add_ons=add_ons)
        _clean_up_project(Path("./new-kedro-project"))
```

You need to clean up the project after creation to fix the coverage issues.